cekit/builders/osbs.py

```diff
@@ -27,40 +27,31 @@
 class OSBSBuilder(Builder):
     """Class representing OSBS builder."""
 
-    def __init__(self, build_engine, target, params=None):
-        if not params:
-            params = {}
-        self._commit_msg = params.get('commit_msg')
-        self._user = params.get('user')
-        self._nowait = params.get('nowait', False)
-        self._release = params.get('release', False)
-        self._target = params.get('target')
-        self._rhpkg_set_url_repos = []
+    def __init__(self, common_params, params):
+        super(OSBSBuilder, self).__init__('osbs', common_params, params)
 
-        self._stage = params.get('stage', False)
+        self._rhpkg_set_url_repos = []
 
-        if params.get('redhat'):
-            if params.get('stage'):
-                self._fedpkg = 'rhpkg-stage'
-                self._koji = 'brew-stage'
+        if config.get('common', 'redhat'):
+            if self.params.get('stage'):
+                self._fedpkg = '/usr/bin/rhpkg-stage'
+                self._koji = '/usr/bin/brew-stage'
```

Comment:
Question - isn't /usr/bin always in the path ? Why does it need to be explicitly specified?

Reply:
Its safer to specify absolute path to the binary. In other cases it may be a security issue. Yes we need to do it in several other places.

Pylint complains about this.

Reply:
Ok.